lighthouse-core/test/results/sample_v2.json

```diff
@@ -1331,28 +1323,37 @@
           {
             "node": {
               "type": "node",
-              "selector": "body > img[src$=\"lighthouse-480x318.jpg\"]:nth-child(5)",
-              "path": "3,HTML,1,BODY,5,IMG",
+              "selector": "img[height=\"\\35 7\"]",
+              "path": "3,HTML,1,BODY,10,IMG",
               "snippet": "<img src=\"lighthouse-480x318.jpg\" width=\"480\" height=\"57\">",
-              "explanation": "Fix any of the following:\n  Element does not have an alt attribute\n  aria-label attribute does not exist or is empty\n  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible\n  Element has no title attribute or the title attribute is empty\n  Element's default semantics were not overridden with role=\"presentation\"\n  Element's default semantics were not overridden with role=\"none\""
+              "explanation": "Fix any of the following:\n  Element does not have an alt attribute\n  aria-label attribute does not exist or is empty\n  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty\n  Element has no title attribute or the title attribute is empty\n  Element's default semantics were not overridden with role=\"presentation\"\n  Element's default semantics were not overridden with role=\"none\""
             }
           },
           {
             "node": {
               "type": "node",
-              "selector": "body > img[src$=\"lighthouse-480x318.jpg\"]:nth-child(6)",
-              "path": "3,HTML,1,BODY,7,IMG",
+              "selector": "img[height=\"\\33 18\"]",
```

Comment:
yeah IMO these bad boys are real regressions :/ 

I guess we could file an upstream issue and hope they improve it soon?

Reply:
> I guess we could file an upstream issue and hope they improve it soon?

Looks like it might have been https://github.com/dequelabs/axe-core/pull/731. One of the comments:

> I'm also not a big fan of dropping the parent selector part. One of the selectors i saw because of that was `b`. Nice and short, but it seems unlikely the user will just be able to guess where that is.

but looks like they were ok with that.